example/linux/ubuntu-22.04-prepare.sh

```diff
@@ -36,6 +36,6 @@ autoinstall:
   late-commands:
     - "echo 'ubuntu ALL=(ALL) NOPASSWD: ALL' > /target/etc/sudoers.d/ubuntu-nopasswd"
 
-  shutdown: "poweroff"
+  shutdown: "reboot"
```

Comment:
We can also remove this key, as "reboot" is the default, but keeping it here explicitly may make sense as well.